index.html

```diff
@@ -172,5 +172,34 @@ <h2>Multi-Post Stories</h2>
             </div>
     
         </section>
+
+        <!-- contact section  -->
+
+        <section class="footer">
+            <div class="contactHeader">
+                
+                <h1>Contact me</h1>
+                <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
+                    I’d love to help with it!
+                </p>
+                <form action="https://formspree.io/f/{form_id}" method="post">
+                    <input type="text" name="name" class="lorem" placeholder="Lorem Ipsum" required></input>
+                    <input type="email" name="email" class="email" placeholder="loremipsum@gmail.com" required></input>
+                    <textarea class="textArea" name="comment" rows="8" placeholder="Write your message here..." maxlength="500" required></textarea>
+                    <button type="submit" class="btnGet">Get in touch</button>
+                    
+                </form>
+            
+            </div>
+
+            
+               
+        </section>
+
```

Comment:
[Optional] New lines are good for clean code. But keep it consistent. Normally, we use only one new line to separate the sections.